extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
can we put this logic in `removeFromFixedLayer`? I suspect this is an a11y issue for all fixed layer items since fixedLayer is a sibling of body and focus needs to be returned to the actual body.